f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -384,6 +386,45 @@ def _delete_persist_profile(self, bigip, vs):
         self._delete_http_cookie_persist_profile(bigip, vs)
         self._delete_source_addr_persist_profile(bigip, vs)
 
+    def _delete_http_profile(self, bigip, vs):
+        payload = {
+            "name": "http_profile_" + vs['name'],
+            "partition": vs['partition'],
+        }
+        super(ListenerManager, self)._delete(
+            bigip, payload, None, None,
+            helper=self.http_profile_helper)
+
+    def _create_http_profile(self, bigip, vs):
+        # the logic is, if the http_profile_file is configured
+        # in the ini file, then
+        # 1) check if the configured file exists or not.
+        # 2) parse the content in the file
+        # 3) call the restful api to create the http_profile for the listener
+        # 4) set the http_profile in the profiles.
+        if self.driver.conf.f5_http_profile_name:
+            # check if the file exists or not.
+            # check the content of the file content
+            LOG.debug("http profile file configured is %s",
+                      self.driver.conf.f5_http_profile_name)
+            # create http profile
+            payload = {
+                "name": "http_profile_" + vs['name'],
+                "partition": vs['partition'],
+                "insertXforwardedFor": "enabled",
```

Comment:
We may not hardcode this property in this commit. Please utilize your config file to define it in your next commit.

Reply:
> We may not hardcode this property in this commit. Please utilize your config file to define it in your next commit.
yes